Jenkinsfile

```diff
@@ -1,8 +1,10 @@
+import java.util.regex.Pattern
+
 def baseConfig = {
     owner = 'ksql'
     slackChannel = '#ksql-alerts'
     ksql_db_version = "0.14.0"  // next version to be released
-    cp_version = "6.1.0-beta201006024150"  // must be a beta version from the packaging build
+    cp_version = "6.1.0-beta201020194508"  // must be a beta version from the packaging build
```

Comment:
Hey @elismaga (and the rest of the tools team since Eli is OOO), this version bump is a problem for us since the previous version `6.1.0-beta201006024150` has been qualified for our release, but not this new one. Do we have options to release with the old version?

Reply:
I'm not familiar with the ksql release process so apologies in advance, but would it be a possibility if we instead cherrypick the nanoversioning rollout changes on top of the old 6.1.0-beta201006024150? Or would qualifying that take just as long as qualifying the newer version?

Reply:
6.1.0-beta201006024150 is a beta packaging build produced by the tools team. Someone from your team can probably comment more authoritatively than I can on how difficult it is to cherry-pick and reproduce beta artifacts?

Reply:
I do not image that would be easy to cherrypick nanoversion change to the old beta version, I guess we could try just add the regex part and removed the version bump part.